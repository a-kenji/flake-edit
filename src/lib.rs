pub mod change;
pub mod diff;
pub mod edit;
pub mod error;
mod git;
pub mod input;
pub mod state;
pub mod walk;
