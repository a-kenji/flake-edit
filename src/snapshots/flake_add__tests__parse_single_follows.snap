---
source: src/lib.rs
expression: state.inputs
---
nixpkgs:
  id: nixpkgs
  flake: true
  url: "github:nixos/nixpkgs/nixos-unstable"
  follows: []
rust-overlay:
  id: rust-overlay
  flake: true
  url: "github:oxalica/rust-overlay"
  follows:
    - Indirect:
        - nixpkgs
        - "\"nixpkgs\""

